### PR TITLE
Js on menu

### PR DESCRIPTION
**Pull request by hasnatali1947**
working on the mobile menu 
- added javaScript on the menu
- added query-selector

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Hasnat, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, which the Pull Request has no description. Kindly provide description for this Pull Request which should have short summaries, describing introduced changes in more detail.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -83,6 +84,56 @@ main {
...
+  .a {
+    text-decoration: none;
+    font-size: 32px;
+    color: #fff;
+  }
+
+  .blur {
+    filter: blur(5px);
+  }
+
```
Comment (rloterh) [inline style.css]: Good job adding mobile menu to your design so far! There are however, few thing we need to improve upon with regards to the mobile menu.
- [x] When a mobile menu item is selected the jumped to section still stays blurred. Let's ensure to remove the overlay and blurred feature once a mobile menu is clicked
- [ ] Let's also ensure that the _hamburger_ and the _close_ button (X) are in the same position when toggled, so as to enhance the UX. 

**After menu item is clicked:**
**Portolio section:**        |  **About section:**        |  **Contact section:**     
:-------------------------:|:-------------------------:|:-------------------------:
Home Page | Home Page | Home Page


**Hamburger and close (X) button not aligned:**
**Portolio section:**        |  **About section:**        
:-------------------------:|:-------------------------:
Home Page | Home Page

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Hasnat, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦  🏃 

## Highlights 👍  
- Menu items jump to the corresponding part of the page when clicked!
- Mobile menu icon is hidden on the desktop screen! 

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Kindly include appropriate instructions in your ReadMe on how to _setup_ and _run_ your app, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯



## Highlights 🔥🔥🔥🔥
- Followed GitHub flow ✅
- No Linters error 🤩🔥💯
- Great working on the functionality🔥🔥
- Professional README 🔥✨💯💯


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**